### PR TITLE
HTCONDOR-1306-gridmanager-tidying

### PR DESCRIPTION
**Pull request by GregThain**
https://opensciencegrid.atlassian.net/browse/HTCONDOR-1306

# HTCondor Pull Request Checklist for internal reviewers

- [ ] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [ ] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [ ] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [ ] Verify that the branch destination of the PR matches the target version of the ticket
- [ ] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [ ] Check that commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: src/condor_gridmanager/boincresource.cpp [hunk 1]
```diff
@@ -320,10 +320,9 @@ bool BoincResource::JoinBatch( BoincJob *job, std::string &batch_name,
 {
 	if ( !batch_name.empty() ) {
 		BoincBatch *batch = NULL;
-		for ( list<BoincBatch *>::iterator batch_itr = m_batches.begin();
-			  batch_itr != m_batches.end(); batch_itr++ ) {
-			if ( (*batch_itr)->m_batch_name == batch_name ) {
-				batch = *batch_itr;
+		for (auto & m_batche : m_batches) {
+			if ( m_batche->m_batch_name == batch_name ) {
+				batch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:325]:
```suggestion
		for (auto & next_batch : m_batches) {
			if ( next_batch->m_batch_name == batch_name ) {
				batch = next_batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 2]
```diff
@@ -354,14 +353,13 @@ bool BoincResource::JoinBatch( BoincJob *job, std::string &batch_name,
 	} else {
 
 		BoincBatch *batch = NULL;
-		for ( list<BoincBatch *>::iterator batch_itr = m_batches.begin();
-			  batch_itr != m_batches.end(); batch_itr++ ) {
+		for (auto & m_batche : m_batches) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:356]:
```suggestion
		for (auto & next_batch : m_batches) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 3]
```diff
@@ -354,14 +353,13 @@ bool BoincResource::JoinBatch( BoincJob *job, std::string &batch_name,
...
 			// Assume all jobs with the same application can go into the
 			// same boinc batch.
 			// But we can't add this job to a batch that's already been
 			// submitted.
-			if ( (*batch_itr)->m_app_name == job->GetAppName() && (*batch_itr)->m_submit_status == BatchUnsubmitted ) {
-				batch = *batch_itr;
+			if ( m_batche->m_app_name == job->GetAppName() && m_batche->m_submit_status == BatchUnsubmitted ) {
+				batch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:362]:
```suggestion
			if ( next_batch->m_app_name == job->GetAppName() && next_batch->m_submit_status == BatchUnsubmitted ) {
				batch = next_batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 4]
```diff
@@ -403,10 +401,9 @@ BoincSubmitResponse BoincResource::Submit( BoincJob *job,
 	}
 
 	BoincBatch *batch = NULL;
-	for ( list<BoincBatch *>::iterator batch_itr = m_batches.begin();
-		  batch_itr != m_batches.end(); batch_itr++ ) {
-		if ( (*batch_itr)->m_batch_name == job->remoteBatchName ) {
-			batch = *batch_itr;
+	for (auto & m_batche : m_batches) {
+		if ( m_batche->m_batch_name == job->remoteBatchName ) {
+			batch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:406]:
```suggestion
	for (auto & next_batch : m_batches) {
		if ( next_batch->m_batch_name == job->remoteBatchName ) {
			batch = next_batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 5]
```diff
@@ -462,14 +459,13 @@ bool BoincResource::BatchReadyToSubmit( BoincBatch *batch, unsigned *delay )
 
 bool BoincResource::JobDone( BoincJob *job )
 {
-	for ( list<BoincBatch *>::iterator batch_itr = m_batches.begin();
-		  batch_itr != m_batches.end(); batch_itr++ ) {
+	for (auto & m_batche : m_batches) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:462]:
```suggestion
	for (auto & batch : m_batches) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 6]
```diff
@@ -462,14 +459,13 @@ bool BoincResource::BatchReadyToSubmit( BoincBatch *batch, unsigned *delay )
...
 
-		if ( (*batch_itr)->m_jobs.find( job ) == (*batch_itr)->m_jobs.end() ) {
+		if ( m_batche->m_jobs.find( job ) == m_batche->m_jobs.end() ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:464]:
```suggestion
		if ( batch->m_jobs.find( job ) == batch->m_jobs.end() ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 7]
```diff
@@ -462,14 +459,13 @@ bool BoincResource::BatchReadyToSubmit( BoincBatch *batch, unsigned *delay )
...
 			continue;
 		}
-		(*batch_itr)->m_jobs_done.insert( job );
-		if ( (*batch_itr)->m_jobs_done == (*batch_itr)->m_jobs ) {
+		m_batche->m_jobs_done.insert( job );
+		if ( m_batche->m_jobs_done == m_batche->m_jobs ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:468]:
```suggestion
		batch->m_jobs_done.insert( job );
		if ( batch->m_jobs_done == batch->m_jobs ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 8]
```diff
@@ -508,14 +504,13 @@ void BoincResource::DoPing( unsigned& ping_delay, bool& ping_complete,
...
 	m_statusBatches.clearAll();
-	for ( std::list<BoincBatch*>::iterator itr = m_batches.begin();
-		  itr != m_batches.end(); itr++ ) {
-		if ( (*itr)->m_submit_status == BatchSubmitted ||
-			 (*itr)->m_submit_status == BatchMaybeSubmitted ) {
-			if ( m_needFullQuery && !(*itr)->m_need_full_query ) {
+	for (auto & m_batche : m_batches) {
+		if ( m_batche->m_submit_status == BatchSubmitted ||
+			 m_batche->m_submit_status == BatchMaybeSubmitted ) {
+			if ( m_needFullQuery && !m_batche->m_need_full_query ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:510]:
```suggestion
	for (auto & batch : m_batches) {
		if ( batch->m_submit_status == BatchSubmitted ||
			 batch->m_submit_status == BatchMaybeSubmitted ) {
			if ( m_needFullQuery && !batch->m_need_full_query ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 9]
```diff
@@ -508,14 +504,13 @@ void BoincResource::DoPing( unsigned& ping_delay, bool& ping_complete,
...
 				continue;
 			}
-			m_statusBatches.append( (*itr)->m_batch_name.c_str() );
+			m_statusBatches.append( m_batche->m_batch_name.c_str() );
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:513]:
```suggestion
			m_statusBatches.append( batch->m_batch_name.c_str() );
```

File: src/condor_gridmanager/boincresource.cpp [hunk 10]
```diff
@@ -552,10 +547,9 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
 			ptr += 15;
 
 			BoincBatch *batch = NULL;
-			for ( std::list<BoincBatch*>::iterator batch_itr = m_batches.begin();
-				  batch_itr != m_batches.end(); batch_itr++ ) {
-				if ( (*batch_itr)->m_batch_name == ptr ) {
-					batch = *batch_itr;
+			for (auto & m_batche : m_batches) {
+				if ( m_batche->m_batch_name == ptr ) {
+					batch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:552]:
```suggestion
			for (auto & next_batch : m_batches) {
				if ( next_batch->m_batch_name == ptr ) {
					batch = next_batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 11]
```diff
@@ -582,15 +576,14 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
 
 		// If this error looks like it would affect a submit command,
 		// notify all jobs in BatchMaybeSubmitted state.
-		for ( std::list<BoincBatch*>::iterator batch_itr = m_batches.begin();
-			  batch_itr != m_batches.end(); batch_itr++ ) {
-			if ( (*batch_itr)->m_submit_status != BatchMaybeSubmitted ||
-				 !(*batch_itr)->m_error_message.empty() ) {
+		for (auto & m_batche : m_batches) {
+			if ( m_batche->m_submit_status != BatchMaybeSubmitted ||
+				 !m_batche->m_error_message.empty() ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:581]:
```suggestion
		for (auto & next_batch : m_batches) {
			if ( next_batch->m_submit_status != BatchMaybeSubmitted ||
				 !next_batch->m_error_message.empty() ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 12]
```diff
@@ -582,15 +576,14 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
...
 				continue;
 			}
-			(*batch_itr)->m_error_message = m_statusGahp->getErrorString();
-			for ( set<BoincJob *>::iterator job_itr = (*batch_itr)->m_jobs.begin();
-				  job_itr != (*batch_itr)->m_jobs.end(); job_itr++ ) {
+			m_batche->m_error_message = m_statusGahp->getErrorString();
+			for ( set<BoincJob *>::iterator job_itr = m_batche->m_jobs.begin();
+				  job_itr != m_batche->m_jobs.end(); job_itr++ ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:586]:
```suggestion
			next_batch->m_error_message = m_statusGahp->getErrorString();
			for ( set<BoincJob *>::iterator job_itr = next_batch->m_jobs.begin();
				  job_itr != next_batch->m_jobs.end(); job_itr++ ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 13]
```diff
@@ -611,15 +604,14 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
...
 		ASSERT( batch_name );
 
 		BoincBatch *batch = NULL;
-		for ( list<BoincBatch *>::iterator batch_itr = m_batches.begin();
-			  batch_itr != m_batches.end(); batch_itr++ ) {
-			if ( (*batch_itr)->m_batch_name == batch_name ) {
-				batch = *batch_itr;
+		for (auto & m_batche : m_batches) {
+			if ( m_batche->m_batch_name == batch_name ) {
+				batch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:614]:
```suggestion
		for (auto & next_batch : m_batches) {
			if ( next_batch->m_batch_name == batch_name ) {
				batch = next_batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 14]
```diff
@@ -637,17 +629,16 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
 				// Batch exists on server
 				// Signal the jobs
 				batch->m_submit_status = BatchSubmitted;
-				for ( set<BoincJob *>::iterator job_itr = batch->m_jobs.begin();
-					  job_itr != batch->m_jobs.end(); job_itr++ ) {
-					(*job_itr)->SetEvaluateState();
+				for (auto m_job : batch->m_jobs) {
+					m_job->SetEvaluateState();
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:633]:
```suggestion
				for (auto job : batch->m_jobs) {
					job->SetEvaluateState();
```

File: src/condor_gridmanager/boincresource.cpp [hunk 15]
```diff
@@ -669,8 +660,8 @@ BoincResource::BatchStatusResult BoincResource::FinishBatchStatus()
 		// If not doing a full query, update all jobs in the batch that
 		// their status is unchanged.
 		if ( batch && !m_doingFullQuery ) {
-			for ( std::set<BoincJob*>::iterator job_itr = batch->m_jobs.begin(); job_itr != batch->m_jobs.end(); job_itr++ ) {
-				(*job_itr)->NewBoincState( (*job_itr)->remoteState.c_str() );
+			for (auto m_job : batch->m_jobs) {
+				m_job->NewBoincState( m_job->remoteState.c_str() );
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:664]:
```suggestion
			for (auto job : batch->m_jobs) {
				job->NewBoincState( job->remoteState.c_str() );
```

File: src/condor_gridmanager/boincresource.cpp [hunk 16]
```diff
@@ -692,22 +683,21 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
 		return;
 	}
 
-	for ( list<BoincBatch*>::iterator batch = m_batches.begin();
-		  batch != m_batches.end(); batch++ ) {
+	for (auto & m_batche : m_batches) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:686]:
```suggestion
	for (auto & batch : m_batches) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 17]
```diff
@@ -692,22 +683,21 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 
-		if ( (*batch)->m_submit_status == BatchMaybeSubmitted ||
-			 (*batch)->m_submit_status == BatchSubmitted ||
-			 (*batch)->m_submit_status == BatchFailed ) {
+		if ( m_batche->m_submit_status == BatchMaybeSubmitted ||
+			 m_batche->m_submit_status == BatchSubmitted ||
+			 m_batche->m_submit_status == BatchFailed ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:690]:
```suggestion
		if ( batch->m_submit_status == BatchMaybeSubmitted ||
			 batch->m_submit_status == BatchSubmitted ||
			 batch->m_submit_status == BatchFailed ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 18]
```diff
@@ -692,22 +683,21 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 			continue;
 		}
 
 		// If we have an active submit command, skip to that batch
 		unsigned this_delay = TIMER_NEVER;
 		if ( m_activeSubmitBatch != NULL ) {
-			if ( *batch != m_activeSubmitBatch ) {
+			if ( m_batche != m_activeSubmitBatch ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:697]:
```suggestion
			if ( batch != m_activeSubmitBatch ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 19]
```diff
@@ -692,22 +683,21 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 				continue;
 			}
-		} else if ( !BatchReadyToSubmit( *batch, &this_delay ) ) {
+		} else if ( !BatchReadyToSubmit( m_batche, &this_delay ) ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:700]:
```suggestion
		} else if ( !BatchReadyToSubmit( batch, &this_delay ) ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 20]
```diff
@@ -717,15 +707,15 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
 		if ( m_activeSubmitBatch == NULL ) {
 
 			// Let's start submitting this batch
-			int rc = m_submitGahp->boinc_submit( (*batch)->m_batch_name.c_str(),
-												 (*batch)->m_jobs );
+			int rc = m_submitGahp->boinc_submit( m_batche->m_batch_name.c_str(),
+												 m_batche->m_jobs );
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:711]:
```suggestion
			int rc = m_submitGahp->boinc_submit( batch->m_batch_name.c_str(),
												 batch->m_jobs );
```

File: src/condor_gridmanager/boincresource.cpp [hunk 21]
```diff
@@ -717,15 +707,15 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 			if ( rc != GAHPCLIENT_COMMAND_PENDING ) {
 				dprintf( D_ALWAYS, "New boinc_submit() didn't return PENDING!?: %s\n", m_submitGahp->getErrorString() );
 				m_submitGahp->purgePendingRequests();
 				// TODO What else should we do?
 			} else {
-				(*batch)->m_submit_status = BatchSubmitting;
-				m_activeSubmitBatch = (*batch);
+				m_batche->m_submit_status = BatchSubmitting;
+				m_activeSubmitBatch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:718]:
```suggestion
				batch->m_submit_status = BatchSubmitting;
				m_activeSubmitBatch = batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 22]
```diff
@@ -734,8 +724,8 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
 			// active submit command, check if it's done
 			// TODO avoid overhead of recreating arguments for 
 			//   already-submitted command
-			int rc = m_submitGahp->boinc_submit( (*batch)->m_batch_name.c_str(),
-												 (*batch)->m_jobs );
+			int rc = m_submitGahp->boinc_submit( m_batche->m_batch_name.c_str(),
+												 m_batche->m_jobs );
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:728]:
```suggestion
			int rc = m_submitGahp->boinc_submit( batch->m_batch_name.c_str(),
												 batch->m_jobs );
```

File: src/condor_gridmanager/boincresource.cpp [hunk 23]
```diff
@@ -745,18 +735,18 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
 
 			if ( rc == 0 ) {
 				// success
-				(*batch)->m_submit_status = BatchSubmitted;
+				m_batche->m_submit_status = BatchSubmitted;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:738]:
```suggestion
				batch->m_submit_status = BatchSubmitted;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 24]
```diff
@@ -745,18 +735,18 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 				daemonCore->Reset_Timer( m_leaseTid, 0 );
 			} else {
 				dprintf( D_ALWAYS, "Failed to submit batch %s: %s\n",
-						 (*batch)->m_batch_name.c_str(),
+						 m_batche->m_batch_name.c_str(),
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:742]:
```suggestion
						 batch->m_batch_name.c_str(),
```

File: src/condor_gridmanager/boincresource.cpp [hunk 25]
```diff
@@ -745,18 +735,18 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 						 m_submitGahp->getErrorString() );
-				(*batch)->m_submit_status = BatchFailed;
-				(*batch)->m_error_message = m_submitGahp->getErrorString();
+				m_batche->m_submit_status = BatchFailed;
+				m_batche->m_error_message = m_submitGahp->getErrorString();
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:745]:
```suggestion
				batch->m_submit_status = BatchFailed;
				batch->m_error_message = m_submitGahp->getErrorString();
```

File: src/condor_gridmanager/boincresource.cpp [hunk 26]
```diff
@@ -745,18 +735,18 @@ dprintf(D_FULLDEBUG,"*** DoBatchSubmits()\n");
...
 			}
 
-			for ( set<BoincJob *>::iterator job = (*batch)->m_jobs.begin();
-				  job != (*batch)->m_jobs.end(); job++ ) {
+			for ( set<BoincJob *>::iterator job = m_batche->m_jobs.begin();
+				  job != m_batche->m_jobs.end(); job++ ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:749]:
```suggestion
			for ( set<BoincJob *>::iterator job = batch->m_jobs.begin();
				  job != batch->m_jobs.end(); job++ ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 27]
```diff
@@ -781,25 +771,24 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
 		return;
 	}
 
-	for ( list<BoincBatch*>::iterator batch = m_batches.begin();
-		  batch != m_batches.end(); batch++ ) {
+	for (auto & m_batche : m_batches) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:774]:
```suggestion
	for (auto & batch : m_batches) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 28]
```diff
@@ -781,25 +771,24 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 
-		if ( (*batch)->m_submit_status != BatchSubmitted ) {
+		if ( m_batche->m_submit_status != BatchSubmitted ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:776]:
```suggestion
		if ( batch->m_submit_status != BatchSubmitted ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 29]
```diff
@@ -781,25 +771,24 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 			continue;
 		}
 
 		// If we have an active lease update command, skip to that batch
-		if ( m_activeLeaseBatch != NULL && *batch != m_activeLeaseBatch ) {
+		if ( m_activeLeaseBatch != NULL && m_batche != m_activeLeaseBatch ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:781]:
```suggestion
		if ( m_activeLeaseBatch != NULL && batch != m_activeLeaseBatch ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 30]
```diff
@@ -781,25 +771,24 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 			continue;
 		}
 
 		if ( m_activeLeaseBatch == NULL ) {
 			// Calculate how long until this batch's lease needs to be updated
-			time_t renew_time = (*batch)->m_lease_time - ((2 * DEFAULT_LEASE_DURATION) / 3);
+			time_t renew_time = m_batche->m_lease_time - ((2 * DEFAULT_LEASE_DURATION) / 3);
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:787]:
```suggestion
			time_t renew_time = batch->m_lease_time - ((2 * DEFAULT_LEASE_DURATION) / 3);
```

File: src/condor_gridmanager/boincresource.cpp [hunk 31]
```diff
@@ -781,25 +771,24 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 			if ( renew_time <= now ) {
 
-				if ( ( (*batch)->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) > now ) {
-					unsigned this_delay = ( (*batch)->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) - now;
+				if ( ( m_batche->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) > now ) {
+					unsigned this_delay = ( m_batche->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) - now;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:791]:
```suggestion
				if ( ( batch->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) > now ) {
					unsigned this_delay = ( batch->m_last_lease_attempt + LEASE_RETRY_INTERVAL ) - now;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 32]
```diff
@@ -810,14 +799,14 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
 
 				// lease needs to be updated
 				time_t new_lease_time = time(NULL) + DEFAULT_LEASE_DURATION;
-				int rc = m_leaseGahp->boinc_set_lease( (*batch)->m_batch_name.c_str(),
+				int rc = m_leaseGahp->boinc_set_lease( m_batche->m_batch_name.c_str(),
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:802]:
```suggestion
				int rc = m_leaseGahp->boinc_set_lease( batch->m_batch_name.c_str(),
```

File: src/condor_gridmanager/boincresource.cpp [hunk 33]
```diff
@@ -810,14 +799,14 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 													   new_lease_time );
 				if ( rc != GAHPCLIENT_COMMAND_PENDING ) {
 					dprintf( D_ALWAYS, "New boinc_set_lease() didn't return PENDING!?: %s\n", m_leaseGahp->getErrorString() );
 					m_leaseGahp->purgePendingRequests();
 					// TODO What else should we do?
 				} else {
-					m_activeLeaseBatch = (*batch);
+					m_activeLeaseBatch = m_batche;
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:809]:
```suggestion
					m_activeLeaseBatch = batch;
```

File: src/condor_gridmanager/boincresource.cpp [hunk 34]
```diff
@@ -830,25 +819,25 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
 			}
 		} else {
 			// active lease command, check if it's done
-			int rc = m_leaseGahp->boinc_set_lease( (*batch)->m_batch_name.c_str(),
+			int rc = m_leaseGahp->boinc_set_lease( m_batche->m_batch_name.c_str(),
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:822]:
```suggestion
			int rc = m_leaseGahp->boinc_set_lease( batch->m_batch_name.c_str(),
```

File: src/condor_gridmanager/boincresource.cpp [hunk 35]
```diff
@@ -830,25 +819,25 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 												   m_activeLeaseTime );
 			if ( rc == GAHPCLIENT_COMMAND_PENDING ) {
 				// do nothing, wait for command to complete
 				break;
 			}
 			m_activeLeaseBatch = NULL;
-			(*batch)->m_last_lease_attempt = time(NULL);
+			m_batche->m_last_lease_attempt = time(NULL);
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:829]:
```suggestion
			batch->m_last_lease_attempt = time(NULL);
```

File: src/condor_gridmanager/boincresource.cpp [hunk 36]
```diff
@@ -830,25 +819,25 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 			if ( rc == 0 ) {
 				// success
-				(*batch)->m_lease_time = m_activeLeaseTime;
-				for ( set<BoincJob *>::iterator job = (*batch)->m_jobs.begin();
-					  job != (*batch)->m_jobs.end(); job++ ) {
+				m_batche->m_lease_time = m_activeLeaseTime;
+				for ( set<BoincJob *>::iterator job = m_batche->m_jobs.begin();
+					  job != m_batche->m_jobs.end(); job++ ) {
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:834]:
```suggestion
				batch->m_lease_time = m_activeLeaseTime;
				for ( set<BoincJob *>::iterator job = batch->m_jobs.begin();
					  job != batch->m_jobs.end(); job++ ) {
```

File: src/condor_gridmanager/boincresource.cpp [hunk 37]
```diff
@@ -830,25 +819,25 @@ dprintf(D_FULLDEBUG,"*** UpdateBoincLeases()\n");
...
 
 					(*job)->UpdateJobLeaseSent( m_activeLeaseTime );
 				}
 			} else {
 				dprintf( D_ALWAYS, "Failed to set lease for batch %s: %s\n",
-						 (*batch)->m_batch_name.c_str(),
+						 m_batche->m_batch_name.c_str(),
```
Comment (JaimeFrey) [inline src/condor_gridmanager/boincresource.cpp:840]:
```suggestion
						 batch->m_batch_name.c_str(),
```

File: src/condor_gridmanager/ec2-gahp-client.cpp [hunk 38]
```diff
@@ -29,8 +29,8 @@ pushStringListBack( std::vector< YourString > & v, StringList & sl ) {
 
 void
 pushVectorBack( std::vector< YourString > & arguments, const std::vector< std::string > & v ) {
-	for( unsigned i = 0; i < v.size(); ++i ) {
-		arguments.emplace_back(v[i] );
+	for(const auto & i : v) {
+		arguments.emplace_back(i );
```
Comment (JaimeFrey) [inline src/condor_gridmanager/ec2-gahp-client.cpp:33]:
```suggestion
	for(const auto & item : v) {
		arguments.emplace_back(item );
```

**Review by JaimeFrey — CHANGES_REQUESTED** [review]
Fix sub-optimal variable names.

**Comment by timtheisen** [comment]
Wasn't boinc one of the things we were getting rid of?

**Comment by GregThain** [comment]
This branch has 30+ commits, most of which don't build, many of which are partial changes to variable names.  I'm closing this PR, and squashing all these commit into a different commit on a different branch and making a PR out of that.

**Pull request closed without merging**